#!/usr/bin/python3


def raise_exception_msg(message=""):
    raise NameError(message)  # Raises a NameError with the given message
